core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -3561,7 +3581,7 @@ private SyncResult syncInodeMetadata(RpcContext rpcContext, LockedInodePath inod
       if (syncPlan.toUpdateMetaData()) {
         // UpdateMetadata is used when a file or a directory only had metadata change.
         // It works by calling SetAttributeInternal on the inodePath.
-        if (ufsFpParsed.isValid()) {
+        if (ufsFpParsed != null && ufsFpParsed.isValid()) {
```

Comment:
When would `ufsFpParsed` be `null`?

Reply:
I added this because intelliJ was complaining that `ufsFpParsed` could be null. Specifically, `Fingerprint.parse(ufsFingerprint);` a few lines above could return null